### PR TITLE
Update docstring for random_noise function

### PR DESCRIPTION
**Pull request by neeraj3029**
## Description

This pull request refers to the issue #5000 

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Review by sciunto — APPROVED** [review]
Thank you!

File: skimage/util/noise.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def random_noise(image, mode='gaussian', seed=None, clip=True, **kwargs):
                       `low_val` is 0 for unsigned images or -1 for signed
                       images.
         - 'speckle'   Multiplicative noise using out = image + n*image, where
-                      n is uniform noise with specified mean & variance.
+                      n is gaussian noise with specified mean & variance.
```
Comment (mkcor) [inline skimage/util/noise.py:30]:
```suggestion
                      n is Gaussian noise with specified mean & variance.
```
so it matches https://github.com/scikit-image/scikit-image/blob/e19fa1f4c19f4df88a12307409efd8d2de1c4836/skimage/util/noise.py#L19-L20

Comment (neeraj3029) [reply skimage/util/noise.py:30]: Sure, on it!

**Comment by mkcor** [comment]
@neeraj3029 the CI fails are not related to this PR.

@sciunto could you please trigger a rerun of the failing CI workflows?

**Comment by emmanuelle** [comment]
Thank you @neeraj3029 and reviewers @sciunto and @mkcor. Merging!

**Pull request merged**